packages/components/src/components/as-time-series-widget/as-time-series-widget.tsx

```diff
@@ -1,16 +1,23 @@
 import { Component, Event, EventEmitter, h, Method, Prop, Watch } from '@stencil/core';
 import { scaleLinear } from 'd3-scale';
 import { event as d3event } from 'd3-selection';
-import { timeFormat, timeFormatDefaultLocale, TimeLocaleDefinition } from 'd3-time-format';
+import {
+  timeFormat,
+  timeFormatDefaultLocale,
+  TimeLocaleDefinition
+} from 'd3-time-format';
 import { icon } from '../../utils/icons';
 import { AxisOptions, HistogramColorRange, HistogramData, HistogramSelection } from '../as-histogram-widget/interfaces';
 import { RenderOptions } from '../as-histogram-widget/types/RenderOptions';
 import {
+  AUTO_FORMAT,
   DEFAULT_BACKGROUND_BAR_COLOR,
-  DEFAULT_BAR_COLOR
+  DEFAULT_BAR_COLOR,
+  DEFAULT_DATE_FORMAT,
```

Comment:
You should be using this as the default value of the prop, I can't make suggestions on unchanged lines :[